contrib/endpoints/src/api_manager/check_security_rules.cc

```diff
@@ -111,38 +64,31 @@ class AuthzChecker : public std::enable_shared_from_this<AuthzChecker> {
              std::function<void(Status status)> continuation);
 
  private:
-  // Helper method that invokes the test firebase service api.
-  void CallTest(const std::string &ruleset_id,
-                std::shared_ptr<context::RequestContext> context,
-                std::function<void(Status status)> continuation);
+  // This method invokes the Firebase TestRuleset API endpoint as well as user
+  // defined endpoints provided by the TestRulesetResponse.
+  void CallNextRequest(std::function<void(Status status)> continuation);
 
   // Parse the respose for GET RELEASE API call
   Status ParseReleaseResponse(const std::string &json_str,
                               std::string *ruleset_id);
```

Comment:
Here is the Google syle guide link that says output variables should be pointers. The ParseReleaseResponse will set the ruleset_id and making it a pointer is a way of saying that it will be modified. It should also be the last parameter.

https://engdoc.corp.google.com/eng/doc/devguide/cpp/styleguide.shtml?cl=head#Function_Parameter_Ordering